### PR TITLE
Check if the sheet is loaded before applying filters

### PR DESCRIPTION
**Pull request by victor0402**
Fix for https://github.com/coopermaruyama/tableau-react/issues/29

### CODE REVIEW

**Comment by coopermaruyama** [comment]
thanks for this will check it out soon. do you have some setting in your editor that is auto formatting? the diff is a bit hard to parse because there are so many changes that just reformat code without changing behavior. also is there a reason tableau is now being loaded in global scope?

**Comment by victor0402** [comment]
> thanks for this will check it out soon. do you have some setting in your editor that is auto formatting? the diff is a bit hard to parse because there are so many changes that just reformat code without changing behavior. also is there a reason tableau is now being loaded in global scope?

Hi @coopermaruyama! 
I was actually doing some testing and I forgot that my `master` was linked to this PR, my bad.
I just updated it.

**Comment by coopermaruyama** [comment]
i have manually applied your change in 2.1.0 to avoid solving conflicts. thanks!
